homeassistant/components/websocket_api/commands.py

```diff
@@ -246,6 +246,7 @@ def handle_ping(hass, connection, msg):
         vol.Optional("entity_ids"): cv.entity_ids,
         vol.Optional("variables"): dict,
         vol.Optional("timeout"): vol.Coerce(float),
+        vol.Optional("rate_limit"): cv.time_period,
```

Comment:
Why would we accept it via the web socket API since users can set it in the template?

Reply:
Since the user will discover the template crashes their system by pasting it in without the rate limit set, and the next time they restart it will start rendering it again, they end up in an unworkable state without setting it via the api before they paste in the problematic template

Reply:
I'm going to remove this since it doesn't make sense anymore with a default rate limit when accessing all states or all states in a domain is now default.